Reference/Debugging/index.md

```diff
@@ -0,0 +1,71 @@
+---
+versionFrom: 8.1.0
+---
+
+# Debugging
+In version Umbraco 8.1+ we have enabled a debugging feature from Microsoft & Visual Studio called SourceLink which allows you to step into the native Umbraco CMS source code.
```

Comment:
Wondering if this is better ?

Microsoft & Visual Studio have introduced a new debugging technology called 'SourceLink' that enables source code debugging of certain .NET assemblies from NuGet - in version Umbraco 8.1+ this feature has been enabled to allow developers to step into the native Umbraco CMS source code.

Reply:
```suggestion
Microsoft & Visual Studio have introduced a new debugging technology called 'SourceLink' that enables source code debugging of certain .NET assemblies from NuGet - in version Umbraco 8.1+ this feature has been enabled to allow developers to step into the native Umbraco CMS source code.
```